src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -47,57 +47,81 @@ public static IEnumerable<IPackageSearchMetadata> GetPackages(ChocolateyConfigur
 
         public static int GetCount(ChocolateyConfiguration configuration, ILogger nugetLogger, IFileSystem filesystem)
         {
-            return execute_package_search(configuration, nugetLogger, filesystem).GetAwaiter().GetResult().Count();
+            var packageRepositories = NugetCommon.GetRemoteRepositories(configuration, nugetLogger, filesystem);
+            var packageRepositoriesResources = NugetCommon.GetRepositoryResources(packageRepositories);
+            string searchTermLower = configuration.Input.to_lower();
+
+            SearchFilter searchFilter = new SearchFilter(configuration.Prerelease);
+            searchFilter.IncludeDelisted = configuration.ListCommand.LocalOnly;
+
+
+            int totalCount = 0;
+            foreach (var repositoryResources in packageRepositoriesResources)
+            {
+                if (repositoryResources.searchResource == null)
+                {
+                    continue;
+                }
+
+                totalCount += repositoryResources.searchResource.SearchCountAsync(searchTermLower, searchFilter, nugetLogger, CancellationToken.None).GetAwaiter().GetResult();

```

Comment:
This need to be updated to account for a couple of specific cases:
- If `--exact` is passed in, in which case the count is one or zero.
- If `--exact --all-versions` is passed in.
- If `--all-versions` is passed in.

Reply:
@TheCakeIsNaOH as we discussed, while these are needed, this initial scope of work was to get the immediate issues with Chocolatey GUI resolved. We can take these as known issues into the first alpha of Chocolatey GUI, and work on correcting this once some more testing has been done. This will tie into the work around verifying that all of the emitted queries from Chocolatey CLI match what was emitted before.